src/ptor/taskScheduler.js

```diff
@@ -2,6 +2,19 @@
 
 var ConfigParser = require('./configParser').ConfigParser;
 
+/**
+ * properties for parallelization and multi browser support:
```

Comment:
The idea is to support only maxInstances e.g. browsers:[{browserName:'chrome',maxInstances:5} would mean that up to 5 parallel sessions are started that drain the execution queue. The queue contains the list of spec.js files.